src/Starcounter.VS.CSBoilerplateApplicationProjectTemplate/Helpers/iHandler.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace $safeprojectname$.Helpers
+{
+    internal interface IHandler
```

Comment:
I'm not sure about this interface, having an internal abstraction just for the sake of registering hooks and handlers. I'd prefer removing that and just statically list relevant classes in main, invoking `Register` on each.